### DISCUSSION TITLE
screenshot where

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [end-4/dots-hyprland#856] [Q&A]
where does it save the screenshots

**Comment by end-4** [comment]
ctrl+prtsc screenshots and saves to to ~/Pictures/Screenshots
other screenshot keybinds just save to clipboard

**Reply by stormyark** [reply]
Is there a way to use `, Print, global, quickshell:regionScreenshot` AND safe the screenshot to `$(xdg-user-dir PICTURES)/Screenshots/Screenshot_"$(date '+%Y-%m-%d_%H.%M.%S')".png`?
